tests/e2e/specs/api/BuildPushRunPodmanContainerAPI.spec.ts

```diff
@@ -130,7 +130,10 @@ oc logs test-hello-pod
 
 	test('Check the project files were imported', async function (): Promise<void> {
 		const projectSection: ViewSection = await projectAndFileTests.getProjectViewSession();
-		expect(await projectAndFileTests.getProjectTreeItem(projectSection, 'Dockerfile.x86_64'), 'Dockerfile not found in the project tree').not.undefined;
+		expect(
+			await projectAndFileTests.getProjectTreeItem(projectSection, 'Dockerfile.x86_64'),
```

Comment:
it doesn't look related to the target issue, IMHO

Reply:
Yeah..it's a result of applying `npm run prettier` Probably the previous commit related to this test wasn't formatted

Reply:
I see. Thanks for details.